course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -172,11 +233,10 @@
         $scope.renderSource = function (data, type, full, meta) {
             return /^.*feed$/.test(data) ? 'Registrar Added' : 'Manually Added';
         },
-        $scope.selectRole = function(roleId, roleName){
-            $scope.selectedrole = roleId;
-            $('#select-role-btn-id').text(roleName);
+        $scope.selectRole = function (role) {
+            $scope.selectedRole = role;
         };
-        $scope.setTitle = function(id) {
+        $scope.setTitle = function (id) {
```

Comment:
Meh, on further googling, I'll revert this.  Looks like my personal preference for space-if-arguments, no-space-if-no-arguments is just a personal preference.